sormas-ui/src/main/java/de/symeda/sormas/ui/utils/components/MultilineLabel.java

```diff
@@ -0,0 +1,17 @@
+package de.symeda.sormas.ui.utils.components;
```

Comment:
Introducing a new class to set label width to 100% seems suspicious, especially since this is there is only one usage of this class.
I would suggest to just set the width for the label.

Reply:
We have this problem in many places in the code. For instance, here: https://github.com/hzi-braunschweig/SORMAS-Project/pull/7341/files. Vaadin guide also suggest to have small components for situations when what they provide is not enough. But in this particular place we can remove the labels completely, the new flow with showing the discard or save changes popup makes them obsolete. @MateStrysewske, what do you think?